anonlink/bloommatcher.py

```diff
@@ -1,8 +1,22 @@
+import inspect
+import warnings
+
 from anonlink._entitymatcher import ffi, lib
 
 __author__ = 'Stephen Hardy, Brian Thorne'
 
 
+def _fname():
+    return inspect.currentframe().f_back.f_code.co_name
+def _deprecation(use_instead=None):
```

Comment:
Consider a decorator which could then use `f.__name__` instead of having to use `inspect`.